less/quicksilver/jakarta/_variables.less

```diff
@@ -388,7 +388,7 @@
 @featured-secondary-text-color:@white;
 
 //== Featured Footer
-@featured-footer-padding:100px 30px;
+@featured-footer-padding:50px 30px;
```

Comment:
How about we simply remove this variable and leverage the default value from our base _variables.less file?

We should update our base value if they are incorrect there.